app.js

```diff
@@ -37,6 +38,20 @@ app.use(express.urlencoded({ extended: false }))
 app.use(cookieParser(process.env.app_session_secret))
 app.use(require('./config/i18n.config').init)
 
+// CSRF setup
+app.use(
+  csrf({
+    cookie: true,
+    signed: true,
+  }),
+)
```

Comment:
Let's stick the generic cookie config that we already have in here as well: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/config/cookie.config.js

Should just go in cleanly using the spread operator `...cookieConfig`.